### DISCUSSION TITLE
配置文档中关于vulkan 10bit 的疑问

### DISCUSSION THREAD

**Discussion by nanachisol** [dyphire/mpv-config#138] [Q&A] [answered]
配置文档在gpu-api注释一节中提出

> vulkan 的 10bit 尚未完全实现 https://github.com/mpv-player/mpv/issues/8554

但使用以下配置测试，可以输出R10G10B10A2，与d3d11一致
```
vo=gpu-next
gpu-api=vulkan
hwdec=auto-copy
dither-depth=auto
```
在vo=gpu下，gpu-api=vulkan与d3d11不同，vulkan是rgb16f，d3d11是rgba16f
opengl下没有测试.

mpv版本信息
```
mpv v0.39.0-432-g7e519397 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
 built on Nov 30 2024 02:04:28
libplacebo version: v7.350.0 (v7.349.0-29-g5ba1376-dirty)
FFmpeg version: N-117959-g4100a2da2
FFmpeg library versions:
   libavcodec      61.26.100
   libavdevice     61.4.100
   libavfilter     10.6.101
   libavformat     61.9.100
   libavutil       59.47.101
   libswresample   5.4.100
   libswscale      8.12.100
```
[gpu-next vulkan.log](https://github.com/user-attachments/files/17965254/gpu-next.vulkan.log)

**Comment by dyphire** [comment, marked as answer]
mpv 在 gpu-api=vulkan 下对于 sdr 内容始终输出 8bit：https://github.com/mpv-player/mpv/pull/13643

**Reply by nanachisol** [reply]
多谢，是我理解错误，原来是指dither一项。不同gpu-api在日志中可以看到明显的区别。

**Comment by nanachisol** [comment]
https://github.com/dyphire/mpv-config/blob/87f6f85b8c24a0061bb5d5d5932ad27787cf33b8/mpv.conf#L15-L20
配置说明中，关于vulkan 10bit支持的内容应该可以删除了。
另外根据粗略测试，同为d3d12va-copy情况下，vulkan性能貌似更优。
edit:进一步在8k分辨率样片下测试，d3d11优于vulkan.
